xml/System.Buffers/StandardFormat.xml

```diff
@@ -125,7 +125,7 @@ Two <xref:System.Buffers.StandardFormat> instances are equal if their <xref:Syst
         <Parameter Name="other" Type="System.Buffers.StandardFormat" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
+        <param name="other">The <see cref="T:System.Buffers.StandardFormat" /> object to compare to current instance.</param>
```

Comment:
```suggestion
        <param name="other">The format to compare to current instance.</param>
```
This follows the usage of [StandardFormat.op_Equality](https://docs.microsoft.com/en-us/dotnet/api/system.buffers.standardformat.op_equality) and avoids duplication of the link to StandardFormat.